files/base.rb

```diff
@@ -169,6 +188,18 @@ def full_description_hash
       'Tip' => tip,
       'Server' => Socket.gethostname,
     }
+
+    if event_description != 'N/A'
+        description_hash['description'] = event_description
```

Comment:
We don't need this, just fill up this hash

Reply:
I wanted it to be optional. In this case, wouldn't it always be sent ?

Reply:
I'm ok with it being always set to N/A